### ISSUE TITLE
Contributors Screen

### ISSUE THREAD

**Issue by Vanethos** [vostpt/mobile-app#17]
**Description**
Create screen with all the contributors for the app

**File Location**
```
- presentation
|__ ui
```

**Requirements**
- From the Github API, get the current contributors information and display it in form of list with:
   - User image
   - User Name
   - User description

**UI**
<img width="364" alt="imagem" src="https://user-images.githubusercontent.com/10728633/63040338-29bf2580-bebd-11e9-86d2-0741927e0392.png">

**Notes**
Expect the following object:
```
{
    "username": "String",
     "name": "String",
     "profilePicture": "String"
      "category": CategoryEnum
}
```

With Category enum having the values:
```
copy,
design,
development
```

**Comment by Vanethos** [comment]
@OldMetalmind We need to analyze this issue:
- How are we getting Github data? In the case of FOGOS, they created an endpoint dedicated to this

- The copy for this screen must be added

**Comment by OldMetalmind** [comment]
Created two issues to take care of the two pending topics
- (Disregard) -> [Issue-42 : List of Contributors ]()
- [Issue-41 : Copy needed](https://github.com/vostpt/mobile-app/issues/41)

**Comment by AmaldevAD** [comment]
do u have api for this particular app?

**Comment by OldMetalmind** [comment]
We have an API for the app itself, however, for this task at hand, we will be loading the contributors from a JSON file in the assets in this first version.
If you desire to contribute to this task, please recommend a JSON structure for contributors that can have different backgrounds, such as designers, copy, coders.

**Comment by Vanethos** [comment]
The issue was updated with the expected JSON object.

**Comment by Lima21** [comment]
I will do this issue

**Comment by Vanethos** [comment]
@Lima21 Any news on this issue?

**Comment by Vanethos** [comment]
closed via #94
